index.html

```diff
@@ -242,40 +191,41 @@ <h2>Photo Gallery</h2>
     <section class="survey">
         <h2>Survey</h2>
         <form method="POST" action="https://formspree.io/xvogdwnk">
-            <label>What is your name?<br>
```

Comment:
Don't use `<br>` to manage spacing.  Use CSS.  This is the reason that we made the form a grid container and gave the labels margins